### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by wuyepabdul**
Added desktop view media queries to style.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -306,9 +234,73 @@ <h1>CONTACT US</h1>
...
+            placeholder="jeremy@gmail.com"
+            required
+          />
+          <textarea
+            class="full-width input-padding"
+            name="message"
+            placeholder="Enter text here"
+            minlength="500"
+            required
+          ></textarea>
```
Comment (Emmyn5600) [inline index.html]: For the UX (user experience) purpose, kindly make sure that the placeholder message are seen by the users. Your form doesn't display it.

File: index.html [hunk 2]
```diff
@@ -306,9 +234,73 @@ <h1>CONTACT US</h1>
               <a href="#"><img src="./icons/em.png" alt="image" /></a>
             </li>
           </ul>
-          <span class="email">wuyepabdul@gmail.com</span>
```
Comment (Emmyn5600) [inline index.html:309]: port42

- [x] Kindly arrange the `About myself` heading text with the paragraph on the same line, heading should be on left and paragraph on right but in the same line

- [x] You can also work on the `Let's connect` heading with the icons to align them under the about myself heading.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @wuyepabdul 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
Changes required :writing_hand:  

      * #### Please check the comment under the review
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (wuyepabdul) [reply index.html]: Thanks for the feedback. The Placeholder text displays  in my browser. Find attached screenshot in readme file

**Review by DanteAlonsoHT — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @wuyepabdul, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**